guides/common/modules/proc_enrolling-server-with-the-ad-server.adoc

```diff
@@ -1,35 +1,32 @@
 [id="Enrolling_Server_with_the_AD_Server_{context}"]
-= Enrolling {ProjectServer} with the AD server
+= Enrolling {ProjectServer} with the Active Directory server
 
-In the {Project} CLI, enroll {ProjectServer} with the Active Directory server.
+In the {Project} CLI, enroll {ProjectServer} with the Active Directory (AD) server.
 
-.Prerequisites
-* GSS-proxy and {nfs-client-package} are installed.
-+
-Installing GSS-proxy and {nfs-client-package}:
+For more information, see link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/8/html-single/integrating_rhel_systems_directly_with_windows_active_directory/index#connecting-rhel-systems-directly-to-ad-using-sssd_integrating-rhel-systems-directly-with-active-directory[Connecting RHEL systems directly to AD using SSSD].
+
+. Enrolling {ProjectServer} with an AD server by using the System Security Services Daemon (SSSD)
+====
+On your {ProjectServer}:
+
+. Install the packages necessary for direct AD integration with SSSD:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# {project-package-install} gssproxy {nfs-client-package}
+# {foreman-maintain} samba-common-tools realmd oddjob oddjob-mkhomedir sssd adcli krb5-workstation
```

Comment:
What does `{foreman-maintain}` expand to?

Reply:
Fixed.